test/templates/ci-cd/maven-ngx-yarn/pom.xml

```diff
@@ -0,0 +1,1030 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
is this file added by mistake? wont this be generated?

Reply:
Some explainations:
- when using CICD with Artifactory options, the CICD sub generator will modify the `pom.xml`. So during the tests, if this file doesn't exist, it will fail
- that's why I added this

Reply:
ok